gender_novels/analysis/gender_pronoun_freq_analysis.py

```diff
@@ -34,21 +44,33 @@ def books_pronoun_freq(corp):
         relative_freq_male[book] = temp_dict['male']
```

Comment:
I'm confused by the size of the dict. Maybe using book as the key makes the whole book the key? I.e. including the whole text.
You could try `relative_freq_male[str(book)] = temp_dict['male']` to see if that reduces the filesize.